### ISSUE TITLE
No icons/images shown in toolbar using Safari Version 8.0.8 on Mac

### ISSUE THREAD

**Issue by Pjino** [liferay/alloy-editor#345]
Using Safari I get this:
![screen shot 2015-09-14 at 17 37 44](https://cloud.githubusercontent.com/assets/14219474/9855622/58fad7ea-5b07-11e5-856c-9b80f932fe22.png)
Is there any workaround?

**Comment by ipeychev** [comment]
Hey @Pjino,

I will have several questions:
1. Which version of AlloyEditor have you installed?
2. How did you install it? Using bower, npm or you've downloaded it?

On my machine Safari opens AlloyEditor 0.5.2 correctly, via bower, npm or downloaded directly, see the screenshots below:

Via bower:
![screen shot 2015-09-14 at 19 19 02](https://cloud.githubusercontent.com/assets/78014/9856556/c205a25c-5b15-11e5-8e07-86474d7548fb.png)

Via npm:
![screen shot 2015-09-14 at 19 22 35](https://cloud.githubusercontent.com/assets/78014/9856607/069538e2-5b16-11e5-8968-e04d0bbf96b8.png)

Downloaded directly:
![screen shot 2015-09-14 at 19 24 20](https://cloud.githubusercontent.com/assets/78014/9856659/604999a0-5b16-11e5-8c43-602d1a899b4d.png)

**Comment by Pjino** [comment]
Hi @ipeychev,

I downloaded the version that comes when using the link in your website.
I'll have a deeper look into this because i get no errors in the console and using Chrome it works...maybe a cache issue. I'l verify and let you know.
Thank you for your quick reply!

**Comment by Pjino** [comment]
It was a cache issue!...
Working fine. Thank you again!

**Comment by ipeychev** [comment]
You are very welcome!

**Comment by jbalsas** [comment]
Hey @Pjino, I think we've seen this on and off, so just a question to see if we can pin it down.

Were you using AlloyEditor before downloading a new version? Was that why you had some cached files?

Maybe there's something going on with the font glyph generation that makes this happen when using and old font with a new version...

**Comment by Pjino** [comment]
Hi @jbalsas . Sorry but only now I saw your comment.
Now that I know it was a cache issue I also think that it was my mistake that is not supposed to happen. I was playing with a new framework (Phoenix made in Elixir) that has Hot code pushes/reload. I can't remember for sure but I think I didn't place the fonts folder in the correct place the first time and the framework published the changes I made to the project without the fonts folder. Then, When I placed it I think I had the cache issue.
Now, this seems to make sense...Nevertheless thank you very much for your concern.
